### ISSUE TITLE
Better looking log page

### ISSUE THREAD

**Issue by mcarton** [ligthyear/clippy-service#21]
The current log page is not… well… letś say pretty :sweat_smile: 
(I mean that page http://clippy.bashy.io/github/Manishearth/rust-clippy/master/log for example)

**Comment by gnunicorn** [comment]
Agreed. I just needed that for debugging and think it is helpful for the developers themselves. But yes, it ain't very pretty at the moment. Made that into a milestone to collect issues about what a status page should/could be like and features it should contain.
